app.js

```diff
@@ -17,16 +17,18 @@ const app = express();
 
 // ℹ️ This function is getting exported from the config folder. It runs most middlewares
 require('./config')(app);
-
 // default value for title local
-const projectName = 'lab-movies-celebrities';
-const capitalized = string => string[0].toUpperCase() + string.slice(1).toLowerCase();
+const projectName = "Luca's Film Create Movies & Celebrities";
 
-app.locals.title = `${capitalized(projectName)}- Generated with Ironlauncher`;
+app.locals.project_title = `${projectName}🚀`;
 
 // 👇 Start handling routes here
 const index = require('./routes/index');
 app.use('/', index);
+const celebritiesCreate = require('./routes/celebrities-routes');
+app.use('/', celebritiesCreate);
+const moviesCreate = require('./routes/movies-routes');
+app.use('/', moviesCreate);
```

Comment:
```suggestion
const celebritiesCreate = require('./routes/celebrities-routes');
app.use('/celebrities', celebritiesCreate);
const moviesCreate = require('./routes/movies-routes');
app.use('/movies', moviesCreate);
```

Así podrías eliminar todos los celebrities y movies iniciales de los otros dos archivos. Digamos que esto de aquí es el "prefijo" de la ruta.